index.js

```diff
@@ -1,153 +1,157 @@
+            'error',
             4,
             {
-                "SwitchCase": 1
+                SwitchCase: 1,
             }
         ],
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
-        "quotes": [
-            "error",
-            "single"
-        ],
-        "one-var": [
-            "error",
-            "never"
-        ],
-        "operator-linebreak": [
-            "error",
-            "before"
-        ],
-        "padded-blocks": [
-            "error",
-            "never"
-        ],
-        "semi": [
-            "error",
-            "always"
-        ],
-        "max-len": [
-            "error",
+        'quotes': ['error', 'single'],
+        'one-var': ['error', 'never'],
+        'operator-linebreak': ['error', 'before'],
+        'padded-blocks': ['error', 'never'],
+        'semi': ['error', 'always'],
+        'max-len': [
+            'error',
             120,
             {
-                "ignorePattern": "^import\\s.+\\sfrom\\s.+;$",
+                ignorePattern: '^import\\s.+\\sfrom\\s.+;$',
             }
         ],
-        "max-statements-per-line": [
-            "error",
+        'max-statements-per-line': [
+            'error',
             {
-                "max": 1
+                max: 1,
             }
         ],
-        "no-multiple-empty-lines": [
-            "error",
+        'no-console': 'error',
+        'no-multiple-empty-lines': [
+            'error',
             {
-                "max": 1
+                max: 1,
             }
         ],
-        "object-curly-spacing": [
-            "error",
-            "always"
-        ],
-        "object-curly-newline": ["error", {
-            "ObjectExpression": {
-                "minProperties": 1
+        'object-curly-spacing': ['error', 'always'],
+        'object-curly-newline': ['error', {
+            ObjectExpression: {
+                minProperties: 1,
             },
-            "ObjectPattern": {
-                "minProperties": 1
+            ObjectPattern: {
+                minProperties: 1,
             },
-            "ImportDeclaration": "never",
-            "ExportDeclaration": {
-                "minProperties": 1
+            ImportDeclaration: 'never',
+            ExportDeclaration: {
+                minProperties: 1,
             }
         }],
-        "object-property-newline": ["error", {
-            "allowAllPropertiesOnSameLine": true
+        'object-property-newline': ['error', {
+            allowAllPropertiesOnSameLine: true,
         }],
-        "space-before-function-paren": [
-            "error",
-            "never"
-        ],
-        "comma-dangle": ["error", {
-            "arrays": "always-multiline",
-            "objects": "always-multiline",
-            "imports": "always-multiline",
-            "exports": "always-multiline",
-            "functions": "never"
+        'space-before-function-paren': ['error', 'never'],
```

Comment:
I'm also not sure about prettier compatibility. Since the goal is to use the configuration company wide it would be missing the point if teams using prettier would have to overwrite everything again.

Reply:
That's right. Didn't think about that 😄

Reply:
I also think the `eslint-config-prettier` fixes all compatibility issues with prettier so we don't need to keep that in mind.